pkg/certificateprovider/azurekeyvault/provider.go

```diff
@@ -0,0 +1,368 @@
+/*
+Copyright The Ratify Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package azurekeyvault
+
+// This class is based on implementation from  azure secret store csi provider
+// Source: https://github.com/Azure/secrets-store-csi-driver-provider-azure/blob/master/pkg/provider/
+import (
+	"context"
+	"crypto/x509"
+	"encoding/pem"
+	"fmt"
+	"path/filepath"
+	"reflect"
+	"regexp"
+	"sort"
+	"strconv"
+	"strings"
+	"time"
+
+	kv "github.com/Azure/azure-sdk-for-go/services/keyvault/v7.1/keyvault"
+	"github.com/Azure/go-autorest/autorest/azure"
+	"github.com/Azure/go-autorest/autorest/date"
+	"github.com/deislabs/ratify/pkg/certificateprovider/azurekeyvault/types"
+	"github.com/sirupsen/logrus"
+
+	"github.com/pkg/errors"
+	"gopkg.in/yaml.v2"
+)
```

Comment:
nit: group imports in the following format

```go
import (
   stdlib

   internal

   external
)
```

Reply:
fixed. thanks